extensions/projection/schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema":"http://json-schema.org/draft-07/schema#",
+  "$id":"schema.json#",
+  "title":"Projection Extension",
+  "description":"STAC Projection Extension to a STAC Item",
+  "allOf":[
+    {
+      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/master/item-spec/json-schema/item.json#/definitions/core"
+    },
+    {
+      "$ref":"#/definitions/proj"
+    }
+  ],
+  "definitions":{
+    "proj":{
+      "properties":{
+        "properties":{
+          "type":"object",
+          "required":[
+            "proj:epsg"
+          ],
+          "properties":{
+            "proj:epsg":{
+              "title":"EPSG code",
+              "type":"integer"
+            },
+            "proj:crs":{
+              "title":"Coordinate Reference System in PROJ format",
+              "type":"string"
+            },
+            "proj:geometry":{
```

Comment:
This seems a bit too basic. Could we derive a JSON Schema from https://geojson.org/schema/Polygon.json.
Maybe we can simply $ref to https://geojson.org/schema/Polygon.json - it doesn't seem to be restricted to a CRS.

Reply:
I'm wondering if geometry in a specific projection might be better served as an asset as a shapefile?
I'm mostly worried that geometry is our biggest property and duplicating it seems like a big lift. Also, there are lots of ways to specify geometries so a specific format would be needed to put it in the metadata to make it readable.

Reply:
Updated with a $ref to Polygon.